app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -73,12 +73,11 @@ class SmokeTest {
         }.enterURLAndEnterToBrowser(defaultWebPage.url) {
         }.openThreeDotMenu {
             verifyThreeDotMainMenuItems()
-            // addOns()
-            /*}.clickAddOnsReportSiteIssue {
+        }.clickAddOnsReportSiteIssue {
             verifyUrl("https://webcompat.com/issues/new")

```

Comment:
To have it working for me, I had to remove the "https://" part

Reply:
okay